components/projectQueryForm.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react'
+import firebase from 'firebase'
+import moment from 'moment-timezone'
+import _ from 'lodash'
+import 'common/initializeFirebase.js'
+import Div from 'components/core/div.js'
+import Button from 'components/core/button.js'
+import styled from 'styled-components'
+
+const OuterContainer = styled(Div)`
+  flex-direction: column;
+  align-items: center;
+`
+
+const FormHeaderContainer = styled(Div)`
+  flex-direction: row;
+  align-items: center;
+`
+
+const CloseQueryAnchor = styled.a`
```

Comment:
Anchor is a weird word? It's a CloseQueryButton or Link (but it's really a button, even if it happens to be a link).